src/core.js

```diff
@@ -66,11 +66,8 @@ const globalContext = typeof window !== 'undefined' ? window : global;
 
 // Creates a single application object asynchronously
 // with a gl context, a camera, a program, a scene, and an event system.
-export async function PhiloGL(canvasId, opt = {}) {
-  opt = {
-    ...DEFAULT_OPTS,
-    ...opt
-  };
+export function PhiloGL(canvasId, opt = {}) {
+  opt = $.merge(DEFAULT_OPTS, opt);
```

Comment:
@ibgreen The spread operator only performs a shallow merge. How about lodash.defaultsDeep?